de/manual/ch_Tools_Assistants.xml

```diff
@@ -70,43 +70,39 @@
-            Konten, die im Dialog <guilabel>Konten wählen</guilabel> markiert wurden, gefunden
-            werden, die mindestens einen Buchungsteil in jedem der ausgewählten Konten enthalten.
-            Wenn die Suche in einem regulären Kontobuch gestartet wird, ist dies das einzige
-            Kriterium, das Buchungsteile von anderen Konten prüft. Wenn Sie also den Dialog
-            <guilabel>Buchungen suchen</guilabel> z.B. in Aktiva:Umlaufvermögen:Girokonto öffnen
-            und alle Buchungen dieses Kontos mit einem Buchungsteil in Ausgaben:Lebensmittel finden
-            wollen, ist dies als Kriterium zu verwenden. Beachten Sie jedoch, dass es sich um eine
-            <emphasis>UND</emphasis>-Suche handelt: Wenn Sie zusätzlich auch nach
-            Ausgaben:Verschiedenes suchen, erhalten Sie <emphasis>nur</emphasis> die Buchungen mit
-            einem Buchungsteil in jedem der beiden Konten und <emphasis>nicht</emphasis> die
-            Buchungen mit nur einem einzigen Buchungsteil.
-          </para>
-        </footnote>
-        Möchten Sie stattdessen das gesamte &app;-Buch durchsuchen, dann starten Sie die Suche
-        entweder in der <emphasis>Kontenübersicht</emphasis>, oder Sie öffnen das
-        &mc.to.gen-jour;.
-      </para>
+      <para>Abhängig davon, wie tief Sie in Ihren Daten abtauchen wollen, werden die folgenden
+        <emphasis>Ausgangssituationen</emphasis> unterschieden:
+        <variablelist>
+          <varlistentry>
+            <term>Ein einzelnes <emphasis>Konto</emphasis> durchsuchen</term>
 
-      <para>Bei den Buchungsfeldern werden nur die Buchungen berücksichtigt, die mit einem Buchungsteil in
-        diesem Register verknüpft sind, während bei den übrigen Buchungen nur die mit dem Konto
-        verbundenen Buchungsteile durchsucht werden, die diesem Konto zugeordnet sind, außer Sie
-        verwenden die Suchfunktion <guilabel>Alle Konten</guilabel>.
+            <listitem>
+              <para>Führen Sie die Suche in der Registerkarte eines Kontos durch, so werden nur die zu diesem Konto
+                zugehörigen Buchungsteile durchsucht.
+              </para>
+            </listitem>
+          </varlistentry>
+
+          <varlistentry>
+            <term>Das gesamte <emphasis>&appname;-Buch</emphasis> durchsuchen</term>
+
+            <listitem>
+              <para>Starten Sie die Suche entweder in der <emphasis>Kontenübersicht</emphasis>, oder in dem geöffneten
+                <xref linkend="general-journal" />, so wird die Suche über alle Konten ausgeführt.
+              </para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
       </para>
 
       <para>Wenn Sie zuvor die Buchungen in einem Register mit &mc.vw.f-by; filtern, dann werden in diesem Konto
         nur die Buchungen durchsucht, die durch den Filter angezeigt werden. Suchen Sie in einem
         gefiltertem Kontobuch oder einem <xref linkend="general-journal" />, werden alle
```

Comment:
Sollten wir vielleicht durchgängig statt `Kontobuch` `Kontenblatt` verwenden? In den Zeiten von Stift und Papier wurde das Journal gewöhnlich in einem gebundenen Buch, aber die Konten auf je einer (oder mehreren, falls voll) Karteikarte geführt.

Das sind so meine Ideen zur Vereinhieitlichung derTerminologie in den Docs, kann also gerne später über beide Docs gemacht werden. Diese Unterscheidung sollte nach Rücksprache mit John auch in das Glossar im Program einfließen.

Reply:
Das Thema `Karteikarte` oder `Reiter` für die einzelnen Seiten eines Dialogs ist ebenfalls nicht einheitlich.

Reply:
Ja, die Ergebnisse der Diskussion von https://lists.gnucash.org/pipermail/gnucash-de/2021-June/012157.html ff. haben wir zwar dann verwendet, aber nicht auf alle Doks angewandt. Sollten wir aber vielleicht mal in einer separaten PR machen.